project/src/app/app.module.ts

```diff
@@ -3,15 +3,17 @@ import { BrowserModule } from '@angular/platform-browser';
 
 import { AppRoutingModule } from './app-routing.module';
 import { AppComponent } from './app.component';
+import {ProductsModule} from "./products/products.module";
 
 @NgModule({
   declarations: [
-    AppComponent
-  ],
-  imports: [
-    BrowserModule,
-    AppRoutingModule
+    AppComponent,
   ],
+    imports: [
+        BrowserModule,
+        AppRoutingModule,
+        ProductsModule
```

Comment:
Delete. You don't need to import it into app.module, because it is a separate module with routing and it will be loaded separately in the routing, and there is no need to put it in import app.module.ts